### PR TITLE
Gun Descriptions including context for Contraband status

### PR DESCRIPTION
**Pull request by Kesiath**
## About the PR
This PR goes through and gives more context to all the guns in game for players, new or old, to be able to quickly see if it should be contraband or not.

## Why / Balance
Tired of hearing the bickering back and forth on whether a gun is contraband or not, people don't actually read space law so hopefully they WILL read the descriptions of the item.

**Changelog**
:cl:
- tweak: Changed descriptions of guns to be clearly contraband or not.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   name: energy gun
   parent: BaseWeaponBattery
   id: WeaponEnergyGun
-  description: "A basic hybrid energy gun with two settings: disable and kill."
+  description: "A basic hybrid energy gun with two settings: disable and kill.  On the handle is a label that says 'for authorized use only.'"
```
Comment (whatston3) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:5]:
```suggestion
  description: "A basic hybrid energy gun with two settings: disable and kill. On the handle is a label that says 'for authorized use only.'"
```
Double space after period here, most descriptions use single space.

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 2]
```diff
@@ -60,7 +60,7 @@
   suffix: DO NOT MAP # Frontier - Add DO NOT MAP, this is the sheriff's gun
   parent: BaseWeaponBatterySmall
   id: WeaponEnergyGunMultiphase
-  description: This is an expensive, modern recreation of an antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time.
+  description: This is an expensive, modern recreation of an antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time.  On the handle is a label that says 'for authorized use only.'
```
Comment (whatston3) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:63]:
```suggestion
  description: This is an expensive, modern recreation of an antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time. On the handle is a label that says 'for authorized use only.'
```
Double space after period here, most descriptions use single space.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 3]
```diff
@@ -389,7 +389,7 @@
   name: x-ray cannon
   parent: BaseWeaponBattery
   id: WeaponXrayCannon
-  description: An experimental weapon that uses concentrated x-ray energy against its target.
+  description: An illegal and experimental weapon that uses concentrated x-ray energy against its target.
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:392]:
```suggestion
  description: An illegal and experimental weapon that uses concentrated x-ray energy against its target.
```
End of line space.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 4]
```diff
@@ -162,7 +162,7 @@
   name: Kammerer
   parent: BaseWeaponShotgun
   id: WeaponShotgunKammerer
-  description: When an old Remington design meets modern materials, this is the result. A favourite weapon of militia forces throughout many worlds. Uses .50 shotgun shells.
+  description: When an old Remington design meets modern materials, this is the result. A civilian grade shotgun, favored ny militia forces throughout many worlds. Uses .50 shotgun shells.
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:165]:
```suggestion
  description: When an old Remington design meets modern materials, this is the result. A civilian grade shotgun, favored by militia forces throughout many worlds. Uses .50 shotgun shells.
```

Minor typo.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 5]
```diff
@@ -184,7 +184,7 @@
   name: sawn-off shotgun
   parent: BaseWeaponShotgun
   id: WeaponShotgunSawn
-  description: Groovy! Uses .50 shotgun shells.
+  description: Groovy! Uses .50 shotgun shells. A modified civilian grade shotgun.
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:187]:
```suggestion
  description: Groovy! Uses .50 shotgun shells. A modified civilian grade shotgun.
```
End of line space.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 6]
```diff
@@ -221,7 +221,7 @@
   name: handmade pistol
   parent: BaseWeaponShotgun
   id: WeaponShotgunHandmade
-  description: Looks unreliable. Uses .50 shotgun shells.
+  description: Looks unreliable, but legal. Uses .50 shotgun shells.
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:224]:
```suggestion
  description: Looks unreliable, but legal. Uses .50 shotgun shells.
```
End of line space.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 7]
```diff
@@ -246,7 +246,7 @@
   parent: BaseWeaponShotgun
   id: WeaponShotgunBlunderbuss
   suffix: Pirate
-  description: Deadly at close range.
+  description: Deadly at close range, an illegal shotgun often sound at the side of a pirate.
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:249]:
```suggestion
  description: Deadly at close range, an illegal shotgun often found at the side of a pirate.
```
Minor typo.

**Review by whatston3 — COMMENTED** [review]
Found a few small, nitpicky inconsistencies.  Otherwise, looks good.

Just to make it explicit, I'm assuming "legal" implies contraband tier 1; "for authorized use", tier 2; and "illegal", tier 3.  Is that right?

**Comment by VividPups** [comment]
I look could you please, do me a favor.. and fix what it said on the enforcer. Upstream called it a clip when its actually a type of mag

**Comment by Kesiath** [comment]
> Found a few small, nitpicky inconsistencies. Otherwise, looks good.
> 
> Just to make it explicit, I'm assuming "legal" implies contraband tier 1; "for authorized use", tier 2; and "illegal", tier 3. Is that right?

Yes, that text on the guns means precisely what you think it does. Also, thanks for the help on this. I should not have done that as tired as I was lol.

**Comment by Kesiath** [comment]
> look could you please, do me a favor.. and fix what it said on the enforcer. Upstream called it a clip when its actually a type of mag

thirty round thirty clip magazine in half a second

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**